src/storage/patch/SparqlUpdatePatchHandler.ts

```diff
@@ -73,6 +77,7 @@ export class SparqlUpdatePatchHandler extends PatchHandler {
     const def = defaultGraph();
     const deletes = op.delete ?? [];
     const inserts = op.insert ?? [];
+    const where: BaseQuad[] = this.isBasicGraphPattern(op.where) ? op.where.patterns : [];
```

Comment:
This is indeed not going to crash anymore, but this is going to allow invalid SPARQL updates to pass through. E.g.:
```sparql
INSERT { ?a ?b ?c }
WHERE {
 { ?a :p ?b }
 UNION
 { ?b :p ?c }
}
```
The above query would be accepted since it is not a BGP and so the `where` would be set to an empty array. In case `where` is defined and it is not a BGP a `NotImplementedHttpError` should be thrown.